db/migrate/20210331123709_add_cache_key_to_exports.rb

```diff
@@ -0,0 +1,6 @@
+class AddCacheKeyToExports < ActiveRecord::Migration[6.1]
+  def change
+    add_column :exports, :key, :text
+    add_index :exports, [:format, :key], unique: true
```

Comment:
Pinaillage : si la colonne est pas trop grande, j'ai l'impression que ça serait bien de rajouter un 'NOT NULL' - quitte à rétro-construire avec de la donnée fake.

Mais si tu penses que ça en vaut pas la peine, ça me va aussi comme ça.

Reply:
Je pense ajouter le not null une fois qu’y a plus de vieux exports. Donc 1h après le deploy. Je préparerai une autre PR.